internal/crypto/key_derivation.go

```diff
@@ -16,6 +19,14 @@ type TLSExporter interface {
 	ComputeExporter(label string, context []byte, keyLength int) ([]byte, error)
 }
 
+func qhkdfExpand(secret []byte, label string, length int) []byte {
+	qlabel := make([]byte, 2+1+5+len(label)+1)
```

Comment:
Where does the final +1 come from?

Reply:
The last 0x00 here: https://quicwg.github.io/base-drafts/draft-ietf-quic-tls.html#one-rtt-secrets. Since Go initializes the slice to 0, we don't need to set it later.

Reply:
Can you leave a comment that this is intentional?